script-menus.rpy

```diff
@@ -73,11 +73,21 @@ label normaltalkmenu_select:
             jump normaltalkmenu
         "Type a question":
             jump normalchatmenu
-        "{b}Anniversary Event{/b}":
+        "{b}Anniversary Event{/b}" if persistent.anniversary:
             if persistent.seen_3yearevent:
                 call screen confirm("Are you sure you want to replay the event?", yes_action=Return, no_action=Jump("ch30_loop"))
             call screen confirm("Initiating this event means you must watch it through before returning to normal gameplay.\nIf you quit at any point you will be returned to the start of the event.\nAre you sure you want to play it now?", yes_action=Jump("ch30_3yearevent"), no_action=Jump("ch30_loop"))
             jump ch30_loop
+        "{b}About the new assets...{/b}" if persistent.art_demo:
```

Comment:
And this
```suggestion
        "{b}About the new assets...{/b}" if art_demo:
```

Reply:
This really should be persistent. It's a setting that is activated for all downloads with the demo on. It must STAY on for that version. Meaning you can't turn the demo off if you are using a demo download.

Reply:
But you set the flag every load. We do not want this value to persist over different versions

Reply:
Since the value is set each load in this version (that's how `define` works), the value will always be set each time you load JN up. As such, loading with this version will always have that flag.

That's why this doesn't have to be persistent. Additionally, if we made it persistent, we need to then clear it on the next version as to not keep filling the persistent with useless data.